sources/packages/web/src/views/aest/StudentApplicationView.vue

```diff
@@ -8,7 +8,40 @@
           params: { studentId },
         }"
         subTitle="Financial Aid Application"
-      >
+        ><template
+          #buttons
+          v-if="
+            applicationDetail.applicationEditStatus ===
+            ApplicationEditStatus.ChangePendingApproval
+          "
+        >
+          <check-permission-role :role="Role.StudentApproveDeclineAppeals">
+            <template #="{ notAllowed }">
+              <v-btn
+                color="primary"
+                variant="outlined"
+                @click="
+                  assessApplicationChangeRequest(
+                    ApplicationEditStatus.ChangeDeclined,
+                  )
+                "
+                :disabled="notAllowed"
+                >Decline change request</v-btn
+              >
+              <v-btn
+                class="ml-2"
+                color="primary"
+                @click="
+                  assessApplicationChangeRequest(
+                    ApplicationEditStatus.ChangedWithApproval,
+                  )
+                "
+                :disabled="notAllowed"
+                >Approve change request</v-btn
```

Comment:
The action button names are not following the AC. If the given button names here in the PR is right, then please have the ACs updated by business.

![image](https://github.com/user-attachments/assets/aab37884-c899-4f1f-a113-d6a130ae7c86)

This is how it looks for current change request. 

![image](https://github.com/user-attachments/assets/62d309bd-106a-4af7-b2ab-1384dfa030a6)